src/main/java/com/iexec/sms/secret/Secret.java

```diff
@@ -99,5 +99,21 @@ public String toString() {
                 ", isEncryptedValue=" + isEncryptedValue +
                 '}';
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Use `@EqualsAndHashCode` lombok annotation instead ?

Reply:
JPA buddy tells me using `@EqualsAndHashCode` is not recommend on entities as it could cause severe performance and memory issues. Please see https://www.jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/ for example.

Reply:
Thanks for the link